lib/fluri.dart

```diff
@@ -199,16 +199,71 @@ class FluriMixin {
     _uri = _uri.replace(queryParameters: queryParameters);
   }
 
+  /// The URI query parameters with support for multi-value params.
+  Map<String, List<String>> get queryParametersAll => _uri.queryParametersAll;
+  set queryParametersAll(Map<String, Iterable<String>> queryParameters) {
```

Comment:
is this setter even necessary?  you could accomplish the same via the existing `queryParameters` setter, though you'd need to do the same `ArgumentError` dance you already do in `setQueryParam`